CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contribution Guidelines
+
+Please help us to make this list better. Your pull requests and suggestions are welcome.
+
+* Make an individual pull request for each suggestion.
+  * If none of the existing categories fit for the tool you are suggesting, you can suggest a new category or suggest to rename an existing one.
+  * You may suggest tools you are developing yourself, or tools developed by others. However, we do not want to include general purpose tools, such as text editors, version control systems, or graphics software.
```

Comment:
Does this make sense? Or can something think of a reason to include tools like that?